packages/UI/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  ChevronsLeft,
+  ChevronsRight,
+  ChevronRight,
+  ChevronLeft,
+} from 'lucide-react';
+import Link from 'next/link';
+import * as React from 'react';
+
+import type { VariantProps } from 'tailwind-variants';
+import { tv } from 'tailwind-variants';
+import { cn } from '../../utils/classesMerger';
+
```

Comment:
Add Jsdocs for pagination, also instructions that using href URL params is necessary for this pagination.

Reply:
good call, thanks!